request.go

```diff
@@ -1,11 +1,40 @@
 package sarama
 
+import (
+	"fmt"
+)
+
 type requestEncoder interface {
 	encoder
 	key() int16
 	version() int16
 }
 
+type requestHeader struct {
```

Comment:
Well... yea. I did not want to touch the production code, hence some duplication.